src/utils.js

```diff
@@ -0,0 +1,93 @@
+// user selects one of these options when first adding an item to a list
+export const radioButtonOptions = [
```

Comment:
per above, recommend moving this to file named app settings, configuration or such